src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import static core.basesyntax.Lottery.getRandomBall;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Ball ball =
+                    getRandomBall(new ColorSupplier().getRandomColor(),
```

Comment:
You are creating a new `ColorSupplier` instance on each loop iteration - that's not good.

Reply:
![image](https://github.com/mate-academy/jv-lottery/assets/68182494/ca5c3ddc-91b9-4b4c-b558-e37897c0c525)
Fixed, now ColorSupplier object creating inside class , but outside the method , so just one ColorSupplier creates